virtual_rainforest/entry_points.py

```diff
@@ -67,31 +69,99 @@ def _parse_command_line_params(
         raise to_raise
 
 
-def vr_run_cli() -> None:
+def install_example_directory(install_dir: Path) -> None:
+    """Install the example directory to a location.
+
+    This function installs the example directory data files and configuration files
+    provided within the package to a selected location. This allows users to look at the
+    simulation directory structure and files more easily and avoids working with the
+    original files inside the package tree.
+
+    The files are installed to a ``vr_example`` directory within the provided install
+    location.
+
+    Args:
+        install_dir: the installation path.
+    """
+    if not install_dir.is_dir():
+        sys.stderr.write("--install_example path is not a valid directory.")
+
+    example_dir = install_dir / "vr_example"
+    if example_dir.exists():
+        sys.stderr.write(
+            "VR example directory already present in --install_example path.\n"
+        )
+
+    copytree(example_data_path, example_dir, ignore=ignore_patterns("__*"))
+
+    sys.stdout.write(f"Example directory created at:\n{example_dir}\n")
```

Comment:
Why not use `print`?

```suggestion
    print(f"Example directory created at:\n{example_dir}\n")
```

Reply:
I guess I was explicitly differentiating between the `stderr` and non-error messages. Is there a cost to explicitly writing to `stdout`?

Reply:
There's not a cost to writing to `stdout` directly, but it's a bit odd from a stylistic point of view. Unless you've overridden the default `print()` function, it'll do the same thing.